packages/quantic/cypress/page-objects/search.ts

```diff
@@ -80,6 +80,27 @@ export function interceptSearchWithError(
   });
 }
 
+export function mockSliceFacetValues(
+  field: string,
+  numberOfFacetResults: number
+) {
+  return cy
+    .intercept('POST', routeMatchers.search, (request) => {
+      request.continue((res) => {
+        const targetFacet = res.body.facets.find(
+          (facet) => facet.field === field
+        );
+        const targetFacetIndex = res.body.facets.indexOf(targetFacet);
```

Comment:
Since `res.body.facets.find()` returns a reference on the object, you could just update the `targetFacet.values` attribute.